packages/babel-plugin-transform-es2015-block-scoping/test/fixtures/general/for-break-continue-return/expected.js

```diff
@@ -1,5 +1,5 @@
 (function () {
-  var _loop = function (i) {
+  var _loop2 = function (i) {
```

Comment:
A change in ordering in block scoping means these variable names are generated in opposite order now, thus breaking the test.